pandas/tests/arithmetic/test_numeric.py

```diff
@@ -299,6 +299,54 @@ def test_ser_div_ser(self, dtype1, dtype2):
         tm.assert_series_equal(result, expected)
         assert not result.equals(second / first)
 
+    @pytest.mark.parametrize(
+        "dtype2",
+        [
+            np.int64,
+            np.int32,
```

Comment:
hmm, don;t' we have a fixture for this?

Reply:
Looks like we have `any_real_dtype`, includes `int` which is not included here.  I copied this list of dtypes from another test in this file.  I'll do a pass to use the fixture consistently in this file.

Reply:
great, yeah can do as a followup